src/GitHub.Exports/Services/Services.cs

```diff
@@ -81,15 +71,9 @@ public static IVsOutputWindowPane OutputWindowPane
             }
         }
 
-        public static DTE Dte
-        {
-            get { return GetGlobalService<DTE, DTE>(); }
-        }
+        public static DTE Dte => GetGlobalService<DTE, DTE>();
 
-        public static DTE2 Dte2
-        {
-            get { return Dte as DTE2; }
-        }
+        public static DTE2 Dte2 => Dte as DTE2;
```

Comment:
@shana: So according to R#, there is no type in the solution that inherits from both `EnvDTE.DTE` and `EnvDTE80.DTE2`. That leads me to believe that the `Dte2` property is always null. Also, I can't find any usages of the `Dte2` property. Do we need this?